### PR TITLE
Treasury now has some Velocity

### PR DESCRIPTION
**Pull request by MrIvanPlays**

### CODE REVIEW

File: platform/velocity/src/main/java/me/lokka30/treasury/plugin/velocity/TreasuryVelocity.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            }
+        }
+        treasuryPlugin = new VelocityTreasuryPlugin(this);
+        TreasuryPlugin.setInstance(treasuryPlugin);
+        treasuryPlugin.loadMessages();
+        treasuryPlugin.loadSettings();
+        TreasuryCommand.register(this);
+
+        UpdateChecker.checkForUpdates();
+        // todo: we need a metrics page so metrics are implemented. lokka30!!!!
```
Comment (MrIvanPlays) [inline platform/velocity/src/main/java/me/lokka30/treasury/plugin/velocity/TreasuryVelocity.java:57]: @lokka30

Comment (lokka30) [reply platform/velocity/src/main/java/me/lokka30/treasury/plugin/velocity/TreasuryVelocity.java:57]: Added metrics page, currently missing additional graphs. Will reply here when I finish adding those

Name: `Treasury-Velocity`
ID: `14651`
URL: [Click](https://bstats.org/plugin/velocity/Treasury-Velocity/14651)

**Comment by lokka30** [comment]
Thank you very much Ivan!

**Comment by lokka30** [comment]
Hi Ivan, is this PR all ready for review now or are you still working on it?

**Comment by MrIvanPlays** [comment]
Ready for review

**Review by lokka30 — APPROVED** [review]
LGTM :)

**Review by Jikoo — APPROVED** [review]
LGTM 👍

**Pull request merged**